source/Calamari.Tests/KubernetesFixtures/KubernetesRunsScriptFixture.cs

```diff
@@ -52,10 +70,214 @@ public void WindowsPowershellKubeCtlScripts()
             SetTestClusterVariables();
```

Comment:
Should we move `SetTestClusterVariables` into `SetUp`? It is used in every test bar one.